### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connect to Server_ColorContrast: The color contrast ratio of data inside "Server Endpoint" form field is lest than 4.5:1

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15808]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.3;#AINotInScope;#ColorContrastAnalyzer;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connect to Server" and press enter
5. Tab to "Server Endpoint" form field.
6. Open Color Contrast Analyzer and its color contrast

### Actual:
The color contrast ratio of data inside "Server Endpoint" form field is 2.6:1 which is less than 4.5:1

### Expected:
The color contrast ratio of data inside "Server Endpoint" form field should be equal to or greater than minimum requirement of 4.5:1

### User Impact:
Low vision users will not be able to view the data properly.

**Comment by Charles-Gagnon** [comment]
@krrish2021Per https://www.w3.org/TR/WCAG21/#contrast-minimum the contrast requirements apply except when `Incidental Text or images of text that are part of an inactive user interface component,`. 

Is there another rule somewhere that is overriding this requirement? The field mentioned in this issue was disabled so by my understanding the contrast requirements don't apply.

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (185)](https://user-images.githubusercontent.com/85863543/150964397-64a94c3a-7c7f-4a47-b561-ef9be14d3fab.png)

**Comment by Harika1702** [comment]
GitHub Tag: #Closed;
